File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResult;

//品牌服务接口
public interface BrandService {

	
	//返回全部列表
	public List<TbBrand> findall();
	
	//返回分页列表
	public PageResult findpage(int pageNum,int pageSize);
	
	//新增品牌
	public void add(TbBrand brand);
	
	//根据id修改品牌
	//查询
	public TbBrand findone(Long id);
	public void updata(TbBrand brand);

	
	//删除品牌
	public void delete(Long [] ids);
	
	//模糊查询
	public PageResult findpage(TbBrand brand,int pageNum,int pageSize);
	
	//品牌下拉
	List<Map> selectOptionList();
}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
//品牌服务层
    app.service('brandService',function($http){
    	//查询所有
    	this.findall=function(){
    		return $http.get('../brand/findall.do');
    	}
    	 //分页查询
    	this.findPage=function(page,rows){	
    		return $http.get('../brand/findpage.do?page='+page+'&rows='+rows);
    	}
    	//新增
    	this.add=function(entity){
    		return $http.post('../brand/add.do',entity);
    	}    	
    	//查询实体
    	this.findOne=function(id){
    		return $http.get('../brand/findone.do?id='+id);
    	}
    	//修改
    	this.update=function(entity){
    		return $http.post('../brand/update.do',entity);
    	}
    	//删除
    	this.del=function(ids){
    		return $http.get('../brand/delete.do?ids='+ids)
    	}
    	//搜素
    	this.search=function(page,rows,searchEntity){
    		return $http.post('../brand/seach.do?page='+page+"&rows="+rows,searchEntity);
    	}
    	//品牌下拉列表
    	this.selectOptionList=function(){
    		return $http.get('../brand/selectOptionList.do');
    	}
    	
    });
    